Source/Core/Core/PrimeHack/Mods/FpsControls.cpp

```diff
@@ -74,6 +74,15 @@ namespace prime {
     pitch = std::clamp(pitch, -1.52f, 1.52f);
   }
 
+  void FpsControls::calculate_pitch_locked() {
```

Comment:
This really should handle regions and games. I understand you only support MP1 Trilogy with this PR, however you haven't defined static addresses for MP1 PAL, and this function will run with the expectation that those static addresses are defined.